src/components/CSnackbar.vue

```diff
@@ -70,11 +71,19 @@ const props = withDefaults(
      * Timeout in milliseconds of this snackbar after which it will be hidden.
      */
     timeout?: number | "infinite" | null;
+
+    /**
+     * The z-index used for the component.
+     *
+     * Defaults to `2500` to be above dialogs (with a z-index of `2400`). (Vuetify default: `2000`)
+     */
+    zIndex?: string | number;
```

Comment:
Why can this be a string?

Reply:
I don't know, I used the same definition as Vuetify: https://vuetifyjs.com/en/api/v-snackbar/#props-z-index